### ISSUE TITLE
[FEATURE] AI Service method parameter annotated with @UserMessage should support template placeholders

### ISSUE THREAD

**Issue by langchain4j** [langchain4j/langchain4j#903]
Reported in https://github.com/langchain4j/langchain4j/issues/805.

Currently, `@UserMessage` annotation placed on the method of AI Service supports placeholders:
```java
interface Greeter {

        @UserMessage("Hi, my name is {{name}}")
        String greet(@V("name") String name);
}
```

Placeholders should be also supported when some String parameter is annotated with `@UserMessage`:
```java
interface Greeter {

        String greet(@UserMessage String userMessage, @V("name") String name);
}
...
String greeting = greeter.greet("Hi, my name is {{name}}", "Klaus");
```

**Comment by edeandrea** [comment]
Should this have been closed when #930 was merged?

**Comment by langchain4j** [comment]
@edeandrea yeah, thanks!
